app/assets/stylesheets/frontend/common.scss

```diff
@@ -0,0 +1,13 @@
+/* the file is place for common style, for example header component or .clearfix and so on */
+html {
+  width: 100%;
+  height: 100%;
```

Comment:
Because i write the login page find the html's width and height just a div element's width and height , i can't make the  div element to center;